OAK-Marketplace-Emotion-Recognition/utils/inference.py

```diff
@@ -0,0 +1,26 @@
+from emotion_recognition_retail import process_frame
+import cv2
+import numpy as np
+from visualization import statistics_update
+
```

Comment:
Please move this file to the root because this is the code that we implement for the post so it's an important part and should be located in a root